Source/IGListCollectionContext.h

```diff
@@ -92,6 +92,25 @@ NS_ASSUME_NONNULL_BEGIN
                                                       atIndex:(NSInteger)index;
 
 /**
+ Dequeues a cell from the UICollectionView reuse pool.
+ 
+ @param nibName           The name of the nib file.
+ @param bundle            The bundle in which to search for the nib file.
+ @param cellClass         The class of the cell you want to dequeue.
+ @param sectionController The section controller requesting this information.
+ @param index             The index of the cell.
+ 
+ @return A cell dequeued from the reuse pool or newly created.
+ 
+ @note This method uses a string representation of the cell class as the identifier.
+ */
+- (__kindof UICollectionViewCell *)dequeueReusableCellFromNibName:(NSString *)nibName
+                                                           bundle:(nullable NSBundle *)bundle
+                                                        cellClass:(Class)cellClass
```

Comment:
You are right. Given that we are returning a `UICollectionViewCell` in either case we don't actually need the class 👍